libopae/plugins/xfpga/sysfs.c

```diff
@@ -778,26 +780,65 @@ struct _fpga_object *alloc_fpga_object(const char *sysfspath, const char *name)
 {
 	struct _fpga_object *obj = calloc(1, sizeof(struct _fpga_object));
```

Comment:
Just a nit - if you're init'ing all of the struct members below, then you can use malloc instead (skips zeroing of members).  On the other hand, if you add new struct members in the future, then they wouldn't be initialized to 0.